web/src/ui_util.ts

```diff
@@ -22,13 +23,30 @@ export function place_caret_at_end(el: HTMLElement): void {
     }
 }
 
-export function replace_emoji_with_text($element: JQuery): void {
+export function process_emoji($element: JQuery): void {
     $element
         .find(".emoji")
         .text(function () {
             if ($(this).is("img")) {
                 return $(this).attr("alt") ?? "";
             }
```

Comment:
Let's add a comment documenting that this first section is for handling custom emoji.